anyway/flask_app.py

```diff
@@ -920,7 +922,7 @@ def index(marker=None, message=None):
     context["iteritems"] = dict.items
     context["hide_search"] = True if request.values.get("hide_search") == "true" else False
     context["embedded_reports"] = get_embedded_reports()
-    context['maps_api_key'] = secrets.get('MAPS_API_KEY')
+    context['maps_api_key'] = secrets.get('MAPS_API_KEY', DEFAULT_MAPS_API_KEY);
```

Comment:
the secrets get functions don't support a default value - https://github.com/hasadna/anyway/blob/dev/anyway/secrets.py#L9

Reply:
also, the api key is not a secret so perhaps it shouldn't be handled by the secrets module

Reply:
You are right!
It is now fixed :)